strapi/strapi
# Add missing keys translations on plugin-upload for pt-BR

### What does it do?

This PR adds translantion for missing keys in the plugin-upload for pt-BR (Brazilian portuguese). It is ready for merge.

### Why is it needed?

Users might visualize english content even though they selected pt-BR as locale for the keys updated with this PR.

### How to test it?

Plugin upload refers to upload features, such as Media Library and Media fields. It is possible to test updating the locale to pt-BR and verifying at the upload modal and media library that all the text is in Brazilian portuguese.

### Related issue(s)/PR(s)

It might be related to [1625 issue](https://github.com/strapi/strapi/issues/1625), although the specific keys are not mentioned in the description and the issue is already closed.
